query1.php

```diff
@@ -239,8 +239,8 @@
 			<!-- footer class to display legal message - sources and copyright statement -->
 			<div class = "footer">
 				<h3 style = "margin-top: 2.5vw; margin-left: 6vw;">Note</h3>
-				<p style = "font-family: 'Noto Sans', sans-serif; font-size: 1.1vw; color: #fff; margin-top: -3.5vw; margin-left: 6vw;">All Album Covers Have Been Sourced From Spotify.<br>Other Images Have Been Sourced From Creative Commons.</p>
-				<p style = "font-family: 'Noto Sans', sans-serif; font-size: 1.1vw; color: #fff; margin-top: -1.2vw; margin-left: 6vw;">&copy Copyright Saahil Prasad 2022, All Rights Reserved.</p>
+				<p style = "font-family: 'Poppins', sans-serif; font-size: 1.1vw; color: #fff; margin-top: -3.5vw; margin-left: 6vw;">All Album Covers Have Been Sourced From Spotify.<br>Other Images Have Been Sourced From Creative Commons.</p>

```

Comment:
Applied the paragraph font - "Poppins" to the copyright statement located in the footer.